hw/ip/aes/data/aes.hjson

```diff
@@ -194,11 +209,81 @@
       '''
     }
   ],
+
+  /////////////////////
+  // Countermeasures //
+  /////////////////////
+
   countermeasures: [
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "CONFIG.SHADOW",
+      desc: "Main control register shadowed."
+    }
+    { name: "CONFIG.SPARSE",
+      desc: "Critical fields in main control register one-hot encoded."
+    }
+    { name: "DATA_REG.SW_UNREADABLE",
+      desc: "Key, IV and data input registers are write only for software."
+    }
+    { name: "DATA_REG.SEC_WIPE",
+      desc: "Key, IV, input/output data and internal state registers can be cleared with pseudo-random data."
+    }
+    { name: "MAIN.FSM.SPARSE",
+      desc: "The main control FSM uses a sparse state encoding."
+    }
+    { name: "MAIN.FSM.MULTIRAIL",
+      desc: "The main control FSM uses multiple, independent logic rails."
+    }
+    { name: "MAIN.FSM.LOCAL_ESC",
+      desc: "The main control FSM moves to a terminal error state upon local escalation."
+    }
+    { name: "MAIN.FSM.GLOBAL_ESC",
+      desc: "The main control FSM moves to a terminal error state upon global escalation."
+    }
+    { name: "CIPHER.FSM.SPARSE",
+      desc: "The cipher core FSM uses a sparse state encoding."
+    }
+    { name: "CIPHER.FSM.MULTIRAIL",
+      desc: "The cipher core FSM uses multiple, independent logic rails."
+    }
+    { name: "CIPHER.FSM.LOCAL_ESC",
+      desc: "The cipher core FSM moves to a terminal error state upon local escalation."
+    }
+    { name: "CTR.FSM.SPARSE",
+      desc: "The CTR mode FSM uses a sparse state encoding."
+    }
+    { name: "CTR.FSM.MULTIRAIL",
+      desc: "The CTR mode FSM uses multiple, independent logic rails."
+    }
+    { name: "CTR.FSM.LOCAL_ESC",
+      desc: "The CTR mode FSM moves to a terminal error state upon local escalation."
+    }
+    { name: "CTR.REDUN",
+      desc: "The AES round counter is protected by a second counter counting in the opposite direction."
+    }
+    { name: "CTRL.SPARSE",
+      desc: "Critical control signals such as handshake and MUX control signals use sparse encodings."
+    }
+    { name: "KEY.SIDELOAD",
+      desc: "1st-order domain-oriented masking of the cipher core."
+    }
+    { name: "KEY.LEAKAGE",
+      desc: "Internal state register cleared with pseudo-random data at the end of the last round."
+    }
+    { name: "KEY.MASKING",
+      desc: "1st-order domain-oriented masking of the cipher core."
+    }
+    //{ name: "DATA_REG.INTEGRITY",
+    //  desc: "Datapath FI hardening."
+    //}
```

Comment:
Could you add a note here to explain why this is included but commented out? ("We're gonna do this, but haven't done it yet" or similar)

Reply:
Done.